docs/ovs-pipeline.md

```diff
@@ -377,34 +377,32 @@ In the future this table may support an additional mode of operations, in which
 it will implement kube-proxy functionality and take care of performing
 load-balancing / DNAT on traffic destined to services.
 
-### CnpEgressRuleTables (45-49)
+### AntreaPolicyEgressRuleTables (45, 49)
 
-For these tables, you will need to keep in mind the CNP [specification](#antrea-network-policy-crd-implementation)
+For these two tables, you will need to keep in mind the Antrea Policy [specification](#antrea-network-policy-crd-implementation)
```

Comment:
technically we are still using an ACNP specification in the example...

Reply:
Yes we are using ACNP as an example, but here I feel like we should mention Antrea NetworkPolicy since the #antrea-networkpolicy-crd-implementation section lures to both CRDs. Let me add explanations as well.